packages/components/inputs/radio-input/src/radio-input.visualroute.js

```diff
@@ -95,7 +95,11 @@ export const component = () => (
         horizontalConstraint="m"
       >
         <RadioInput.Option value="apples">
-          <GreenBox />
+          <GreenBox>
```

Comment:
Added this long text because it gives a better perception about the container size, which is supposed to contain text.